app/models/submission.rb

```diff
@@ -190,6 +190,51 @@ def check_file_properties(metadata, sort)
     { sort => errors }
   end
 
+  def self.bulk_corrections!(tutorial, assignment, files)
+    submissions = Submission.where(tutorial: tutorial,
+                                   assignment: assignment).proper
+    report = { successful_saves: [], submissions: submissions.size,
+    					 invalid_filenames: [], invalid_id: [], in_subfolder: [],
+               no_decision: [], rejected: [], invalid_file: [] }
+    tmp_folder = Dir.mktmpdir
+    begin
+      files.each do |file_shrine|
+      #Zip::File.open(zipfile) do |zip_file|
+      #  zip_file.each do |entry|
```

Comment:
The comments can probably be removed as they represent outdated code (?). They seem rather confusing to me.